beanquery/shell.py

```diff
@@ -157,24 +158,23 @@ def get_pager(self):
         if self.is_interactive:
             return pager.ConditionalPager(self.vars.get('pager', None),
                                           minlines=misc_utils.get_screen_height())
-        else:
-            file = (codecs.getwriter("utf-8")(sys.stdout.buffer)
-                    if hasattr(sys.stdout, 'buffer') else
-                    sys.stdout)
-            return pager.flush_only(file)
+        file = (codecs.getwriter("utf-8")(sys.stdout.buffer)
```

Comment:
Unnecessary "else" after "return" (no-else-return)

Reply:
The new code does not do the same thing as the old.

Reply:
BTW,  unrelated to your change but that code: 

file = (codecs.getwriter("utf-8")(sys.stdout.buffer)

has been causing me problems in Python3. I think there's a better way to do this, setting locale globally.
I'd love to get rid of that eventually.

Reply:
Well, then code was conditionally broken before, and is always broken now. And it fails if the local encoding is not utf8. I don't get what the code was supposed to do, but I am sure it is not correct now.